src/pages/ItemDetail/components/ItemDetailMap.jsx

```diff
@@ -0,0 +1,368 @@
+    },
+    {
+      title: "서초구6",
+      lat: 37.4600336141905,
+      lng: 127.036107243148,
+    },
+    {
+      title: "서초구7",
+      lat: 37.4473433948375,
+      lng: 127.047311765425,
+    },
+    {
+      title: "서초구8",
+      lat: 37.4556862958823,
+      lng: 127.081314291898,
+    },
+    {
+      title: "서초구9",
+      lat: 37.4473238947344,
+      lng: 127.082751771423,
+    },
+    {
+      title: "서초구10",
+      lat: 37.4381148712172,
+      lng: 127.052368194895,
+    },
+    {
+      title: "영등포구1",
+      lat: 37.5254854877886,
+      lng: 126.9160476586,
+    },
+    {
+      title: "영등포구2",
+      lat: 37.5216703411611,
+      lng: 126.923563253669,
+    },
+    {
+      title: "영등포구3",
+      lat: 37.5142088594791,
+      lng: 126.909177485966,
+    },
+    {
+      title: "영등포구4",
+      lat: 37.5318001325465,
+      lng: 126.914134212925,
+    },
+    {
+      title: "종로구1",
+      lat: 37.569049716855,
+      lng: 126.978957247294,
+    },
+    {
+      title: "종로구2",
+      lat: 37.5758436331757,
+      lng: 126.981423187624,
+    },
+    {
+      title: "종로구3",
+      lat: 37.5873043352259,
+      lng: 126.981827937423,
+    },
+    {
+      title: "종로구4",
+      lat: 37.5912250415967,
+      lng: 126.998488412696,
+    },
+    {
+      title: "종로구5",
+      lat: 37.5913688435124,
+      lng: 126.990698341888,
+    },
+    {
+      title: "종로구6",
+      lat: 37.5925916422733,
+      lng: 126.973759044182,
+    },
+    {
+      title: "마포구1",
+      lat: 37.5655690872777,
+      lng: 126.901967305453,
+    },
+    {
+      title: "마포구2",
+      lat: 37.5609653765709,
+      lng: 126.914015867106,
+    },
+    {
+      title: "마포구3",
+      lat: 37.5588733281676,
+      lng: 126.911664158239,
+    },
+    {
+      title: "마포구4",
+      lat: 37.5519453513449,
+      lng: 126.901170343002,
+    },
+    {
+      title: "마포구5",
+      lat: 37.5491448251063,
+      lng: 126.914753695664,
+    },
+    {
+      title: "마포구6",
+      lat: 37.5515990281117,
+      lng: 126.919730270907,
+    },
+    {
+      title: "마포구7",
+      lat: 37.5547724127038,
+      lng: 126.922533540588,
+    },
+    {
+      title: "마포구8",
+      lat: 37.547349372171,
+      lng: 126.940052902236,
+    },
+    {
+      title: "마포구9",
+      lat: 37.5401743126707,
+      lng: 126.934220094085,
+    },
+    {
+      title: "마포구10",
+      lat: 37.5413319812306,
+      lng: 126.942637586044,
+    },
+    {
+      title: "마포구11",
+      lat: 37.5472450081021,
+      lng: 126.947974251826,
+    },
+    {
+      title: "마포구12",
+      lat: 37.5499865950416,
+      lng: 126.95412852726,
+    },
+  ];
+
+  useEffect(() => {
+    mapscript();
+  }, []);
+
+  const mapscript = () => {
+    let container = document.getElementById("map");
+    let options = {
+      center: new kakao.maps.LatLng(37.50521452144289, 126.98323895254065),
+      level: 5,
+    };
+
+    //map
+    const map = new kakao.maps.Map(container, options);
+
+    positions.forEach(e => {
```

Comment:
매개변수를 명확하게 써주는 게 좋을 것 같습니다.
```suggestion
    positions.forEach(position => {
```